Tests/fixtures/podcasting.yml

```diff
@@ -0,0 +1,9536 @@
+    platform: ""
+  client:
+    type: mobile app
+    name: Podyssey
+    version: "1"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'ProCast/1 CFNetwork/1240.0.4 Darwin/20.6.0'
+  os:
+    name: iOS
+    version: "14.7"
+    platform: ""
+  client:
+    type: mobile app
+    name: Procast
+    version: "1"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Procast (iOS)'
+  os:
+    name: iOS
+    version: ""
+    platform: ""
+  client:
+    type: mobile app
+    name: Procast
+    version: ""
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Procast/1.6.7 (com.podflitzer.android; build:16700; Android 10)'
+  os:
+    name: Android
+    version: "10"
+    platform: ""
+  client:
+    type: mobile app
+    name: Procast
+    version: "1.6.7"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'ProCast/2.6.7 (de.dsa.Cuts; build:1; iOS 16.5.1) Alamofire/4.9.1'
+  os:
+    name: iOS
+    version: "16.5.1"
+    platform: ""
+  client:
+    type: mobile app
+    name: Procast
+    version: "2.6.7"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'PugpigBolt 2.29.3 (iPhone, iOS 16.1.1) on phone (model iPhone12,1)'
+  os:
+    name: iOS
+    version: "16.1.1"
+    platform: ""
+  client:
+    type: mobile app
+    name: PugPig Bolt
+    version: "2.29.3"
+  device:
+    type: smartphone
+    brand: Apple
+    model: 'iPhone 11'
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'PugpigBolt 3.8.10 (samsung, Android 13) on phone (model SM-G998U)'
+  os:
+    name: Android
+    version: "13"
+    platform: ""
+  client:
+    type: mobile app
+    name: PugPig Bolt
+    version: "3.8.10"
+  device:
+    type: smartphone
+    brand: Samsung
+    model: 'Galaxy S21 Ultra 5G'
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'GetPodcast 5.6.22 (iPhone; iPhone OS 16.4; de_DE)'
+  os:
+    name: iOS
+    version: "16.4"
+    platform: ""
+  client:
+    type: mobile app
+    name: radio.de
+    version: "5.6.22"
+  device:
+    type: smartphone
+    brand: Apple
+    model: 'iPhone'
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'radio.de/5.10.3.0 (Linux;Android 13) ExoPlayerLib/2.18.2'
+  os:
+    name: Android
+    version: "13"
+    platform: ""
+  client:
+    type: mobile app
+    name: radio.de
+    version: "5.10.3.0"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'GetPodcast/5.10.5.3 (Linux; Android 13; de_DE) ExoPlayerLib/2.18.2'
+  os:
+    name: Android
+    version: "13"
+    platform: ""
+  client:
+    type: mobile app
+    name: radio.de
```

Comment:
GetPodcast is a separate app (https://play.google.com/store/apps/details?id=de.radio.getpodcast&hl=en&gl=US) from radio.de (https://play.google.com/store/apps/details?id=de.radio.android&hl=de) even though it is from same developer, so the name should be "GetPodcast"